repos.yaml

```diff
@@ -631,6 +631,20 @@
       url: 'https://pypi.python.org/pypi/{name}/'
   tags: [ all, preproduction, fastfetch ]
 
+- name: rubygem
+  desc: RubyGems
+  family: rubygem
+  sources:
+    - name: latest_specs.4.8
+      fetcher: File
+      parser: RubyGem
+      url: https://api.rubygems.org/latest_specs.4.8.gz
+      compression: gz
+  links:
+    - desc: RubyGem page
+      url: 'https://www.rubygems.org/gems/{name}/'
+  tags: [ all, have_testdata, demo, production, fastfetch ]
```

Comment:
Just add ```shadow: true``` for it as it's done for CPAN. This way the packages which are only present in this repo **only** won't show up in the metapackages index. If they are present in some other repo, they will show up. A nice way to hide packages we can't compare to anything.